tests/sentry/integrations/utils/test_code_mapping.py

```diff
@@ -129,6 +129,17 @@ def test_raises_unsupported(self):
             with pytest.raises(UnsupportedFrameFilename):
                 FrameFilename(filepath)
 
+    def test_straight_path_prefix(self):
```

Comment:
I think you can put this test outside of this class and use pytest's parametrization:
https://docs.pytest.org/en/6.2.x/parametrize.html

Please give it a try. If it doesn't work we can merge it as-is.